chain/store/weight.go

```diff
@@ -11,6 +11,8 @@ import (
 )
 
 var zero = types.NewInt(0)
+var wRatio_num = build.WRatioNum
```

Comment:
no point redeclaring these here, lets just reference the ones from the build package

Reply:
That's fine. I just wanted to make the wording consistent with the comments in the function.